packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -663,7 +671,7 @@ export class RunningSummarizer implements IDisposable {
 						this.mc.logger.sendPerformanceEvent({
 							eventName: "SummarizeAttemptDelay",
 							duration: delaySeconds,
-							summaryNackDelay: overrideDelaySeconds !== undefined,
+							summaryDelay: overrideDelaySeconds !== undefined,
```

Comment:
While we're changing property names let's move to details model?  Unless it's too disruptive relative to other events (.e.g moving `summarizeProps` off the top level could make querying across events a pain)